compose/parallel.py

```diff
@@ -26,6 +28,18 @@
 STOP = object()
 
 
+def get_configured_limit():
+    limit = Environment.from_command_line({'COMPOSE_PARALLEL_LIMIT': None})['COMPOSE_PARALLEL_LIMIT']
```

Comment:
While I get why you're doing it like this, I think we're definitely looking to be able to set this value in the `.env` file as well.

Reply:
I'd suggest looking at how we went with implementing the `--no-ansi` option in #5093 for guidance.